Framework/PythonInterface/mantid/kernel/src/Exports/ArrayProperty.cpp

```diff
@@ -27,6 +28,11 @@ namespace {
 using return_cloned_numpy =
     return_value_policy<Policies::VectorRefToNumpy<Converters::Clone>>;
 
+// Call the dtype helper function
+template <typename type> std::string dtype(ArrayProperty<type> &self) {
+  return Mantid::PythonInterface::Converters::dtype(self);
```

Comment:
Minor thing: Can we just use `Converters::dtype(self);` as we've already aliased the namespace on line 21 above.

Reply:
No problem, I'll change that!